src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -342,6 +343,11 @@ def report_timeout(self):
                     "*** TEARDOWN called due to TIMEOUT: "
                     "%s second timeout exceeded ***", str(self.timeout))
                 self.log.info("test execution has been terminated by avocado")
+                # dump engines ULT stacks upon test timeout
+                if self.dumped_engines_stacks is False:
```

Comment:
(pylint-access-member-before-definition) Access to member 'dumped_engines_stacks' before its definition line 348